components/popover/index.js

```diff
@@ -25,6 +25,17 @@ const { ESCAPE } = keycodes;
  */
 const ARROW_OFFSET = 20;
 
+function getScrollParents( node ) {
+	if ( ! node ) {
+		return [];
+	}
+	const parentScrollables = getScrollParents( node.parentNode );
+	if ( node.scrollHeight > node.clientHeight ) {
```

Comment:
Thinking this doesn't solve the issue if the parent has no scrollbars yet unless we add content later.